*** Starting uWSGI 2.0.18 (64bit) on [Sun May 31 13:57:24 2020] ***
compiled with version: 7.5.0 on 31 May 2020 05:43:50
os: Linux-4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018
nodename: python
machine: x86_64
clock source: unix
detected number of CPU cores: 4
current working directory: /home/ubuntu/Desktop/meiduo_new/meiduo_mall/meiduo_mall
writing pidfile to uwsgi.pid
detected binary path: /home/ubuntu/.virtualenvs/meiduos_mall/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/Desktop/meiduo_new/meiduo_mall
your processes number limit is 29423
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 192.168.107.188:8001 fd 3
Python version: 3.6.9 (default, Apr 18 2020, 01:56:04)  [GCC 8.4.0]
Set PythonHome to /home/ubuntu/.virtualenvs/meiduos_mall/
Python main interpreter initialized at 0x557a656cd170
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x557a656cd170 pid: 31332 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 31332)
spawned uWSGI worker 1 (pid: 31334, cores: 2)
spawned uWSGI worker 2 (pid: 31335, cores: 2)
spawned uWSGI worker 3 (pid: 31337, cores: 2)
spawned uWSGI worker 4 (pid: 31339, cores: 2)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Sun May 31 14:08:13 2020] ***
compiled with version: 7.5.0 on 31 May 2020 05:43:50
os: Linux-4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018
nodename: python
machine: x86_64
clock source: unix
detected number of CPU cores: 4
current working directory: /home/ubuntu/Desktop/meiduo_new/meiduo_mall/meiduo_mall
writing pidfile to uwsgi.pid
detected binary path: /home/ubuntu/.virtualenvs/meiduos_mall/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/Desktop/meiduo_new/meiduo_mall/meiduo_mall
your processes number limit is 29423
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 192.168.107.188:8001 fd 3
Python version: 3.6.9 (default, Apr 18 2020, 01:56:04)  [GCC 8.4.0]
Set PythonHome to /home/ubuntu/.virtualenvs/meiduos_mall/
Python main interpreter initialized at 0x55c5a306a190
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
failed to open python file meiduo_mall/wsgi.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 31683)
spawned uWSGI worker 1 (pid: 31684, cores: 2)
spawned uWSGI worker 2 (pid: 31685, cores: 2)
spawned uWSGI worker 3 (pid: 31686, cores: 2)
spawned uWSGI worker 4 (pid: 31687, cores: 2)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Sun May 31 14:11:52 2020] ***
compiled with version: 7.5.0 on 31 May 2020 05:43:50
os: Linux-4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018
nodename: python
machine: x86_64
clock source: unix
detected number of CPU cores: 4
current working directory: /home/ubuntu/Desktop/meiduo_new/meiduo_mall/meiduo_mall
writing pidfile to uwsgi.pid
detected binary path: /home/ubuntu/.virtualenvs/meiduos_mall/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/Desktop/meiduo_new/meiduo_mall/meiduo_mall
your processes number limit is 29423
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 192.168.107.188:8001 fd 3
Python version: 3.6.9 (default, Apr 18 2020, 01:56:04)  [GCC 8.4.0]
Set PythonHome to /home/ubuntu/.virtualenvs/meiduos_mall/
Python main interpreter initialized at 0x55933cea5190
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
failed to open python file meiduo_mall/wsgi.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 32019)
spawned uWSGI worker 1 (pid: 32020, cores: 2)
spawned uWSGI worker 2 (pid: 32021, cores: 2)
spawned uWSGI worker 3 (pid: 32022, cores: 2)
spawned uWSGI worker 4 (pid: 32023, cores: 2)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Sun May 31 14:14:45 2020] ***
compiled with version: 7.5.0 on 31 May 2020 05:43:50
os: Linux-4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018
nodename: python
machine: x86_64
clock source: unix
detected number of CPU cores: 4
current working directory: /home/ubuntu/Desktop/meiduo_new/meiduo_mall/meiduo_mall
writing pidfile to uwsgi.pid
detected binary path: /home/ubuntu/.virtualenvs/meiduos_mall/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/Desktop/meiduo_new/meiduo_mall
your processes number limit is 29423
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 192.168.107.188:8001 fd 3
Python version: 3.6.9 (default, Apr 18 2020, 01:56:04)  [GCC 8.4.0]
Set PythonHome to /home/ubuntu/.virtualenvs/meiduos_mall/
Python main interpreter initialized at 0x5565e3187170
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x5565e3187170 pid: 32577 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 32577)
spawned uWSGI worker 1 (pid: 32579, cores: 2)
spawned uWSGI worker 2 (pid: 32581, cores: 2)
spawned uWSGI worker 3 (pid: 32582, cores: 2)
spawned uWSGI worker 4 (pid: 32584, cores: 2)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Sun May 31 14:46:41 2020] ***
compiled with version: 7.5.0 on 31 May 2020 05:43:50
os: Linux-4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018
nodename: python
machine: x86_64
clock source: unix
detected number of CPU cores: 4
current working directory: /home/ubuntu/Desktop/meiduo_new/meiduo_mall/meiduo_mall
writing pidfile to uwsgi.pid
detected binary path: /home/ubuntu/.virtualenvs/meiduos_mall/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/Desktop/meiduo_new/meiduo_mall
your processes number limit is 29423
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 192.168.107.188:8001 fd 3
Python version: 3.6.9 (default, Apr 18 2020, 01:56:04)  [GCC 8.4.0]
Set PythonHome to /home/ubuntu/.virtualenvs/meiduos_mall/
Python main interpreter initialized at 0x564f398b8170
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x564f398b8170 pid: 37624 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 37624)
spawned uWSGI worker 1 (pid: 37627, cores: 2)
spawned uWSGI worker 2 (pid: 37629, cores: 2)
spawned uWSGI worker 3 (pid: 37630, cores: 2)
spawned uWSGI worker 4 (pid: 37632, cores: 2)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
[pid: 37632|app: 0|req: 1/1] 127.0.0.1 () {44 vars in 767 bytes} [Sun May 31 06:49:47 2020] GET /carts/simple/ => generated 803 bytes in 335 msecs (HTTP/1.1 200) 6 headers in 216 bytes (1 switches on core 0)
[pid: 37632|app: 0|req: 2/2] 127.0.0.1 () {44 vars in 767 bytes} [Sun May 31 06:50:22 2020] GET /carts/simple/ => generated 803 bytes in 12 msecs (HTTP/1.1 200) 6 headers in 216 bytes (1 switches on core 1)
[pid: 37627|app: 0|req: 1/3] 127.0.0.1 () {44 vars in 767 bytes} [Sun May 31 06:50:38 2020] GET /carts/simple/ => generated 803 bytes in 260 msecs (HTTP/1.1 200) 6 headers in 216 bytes (1 switches on core 1)
[pid: 37629|app: 0|req: 1/4] 127.0.0.1 () {44 vars in 762 bytes} [Sun May 31 06:50:39 2020] GET /carts/ => generated 918 bytes in 292 msecs (HTTP/1.1 200) 6 headers in 216 bytes (1 switches on core 1)
[pid: 37629|app: 0|req: 3/5] 127.0.0.1 () {44 vars in 796 bytes} [Sun May 31 06:50:42 2020] GET /browse_histories/ => generated 1447 bytes in 45 msecs (HTTP/1.1 200) 6 headers in 217 bytes (1 switches on core 1)
[pid: 37629|app: 0|req: 3/6] 127.0.0.1 () {44 vars in 772 bytes} [Sun May 31 06:50:42 2020] GET /info/ => generated 491 bytes in 316 msecs (HTTP/1.1 200) 6 headers in 216 bytes (1 switches on core 0)
[pid: 37632|app: 0|req: 3/7] 127.0.0.1 () {44 vars in 774 bytes} [Sun May 31 06:50:47 2020] GET /areas/ => generated 1772 bytes in 67 msecs (HTTP/1.1 200) 6 headers in 209 bytes (1 switches on core 0)
[pid: 37630|app: 0|req: 1/8] 127.0.0.1 () {44 vars in 782 bytes} [Sun May 31 06:50:47 2020] GET /addresses/ => generated 811 bytes in 295 msecs (HTTP/1.1 200) 6 headers in 216 bytes (1 switches on core 1)
[pid: 37632|app: 0|req: 4/9] 127.0.0.1 () {44 vars in 788 bytes} [Sun May 31 06:50:53 2020] GET /areas/130000/ => generated 701 bytes in 4 msecs (HTTP/1.1 200) 6 headers in 208 bytes (1 switches on core 1)
[pid: 37627|app: 0|req: 2/10] 127.0.0.1 () {44 vars in 788 bytes} [Sun May 31 06:50:54 2020] GET /areas/130500/ => generated 957 bytes in 6 msecs (HTTP/1.1 200) 6 headers in 208 bytes (1 switches on core 0)
[pid: 37627|app: 0|req: 3/11] 127.0.0.1 () {44 vars in 762 bytes} [Sun May 31 06:50:57 2020] GET /carts/ => generated 918 bytes in 10 msecs (HTTP/1.1 200) 6 headers in 216 bytes (1 switches on core 1)
[pid: 37627|app: 0|req: 4/12] 127.0.0.1 () {46 vars in 755 bytes} [Sun May 31 06:51:06 2020] OPTIONS /login/ => generated 0 bytes in 1 msecs (HTTP/1.1 200) 8 headers in 426 bytes (1 switches on core 0)
[pid: 37627|app: 0|req: 5/13] 127.0.0.1 () {48 vars in 872 bytes} [Sun May 31 06:51:06 2020] POST /login/ => generated 27 bytes in 113 msecs (HTTP/1.1 200) 8 headers in 455 bytes (1 switches on core 1)
[pid: 37629|app: 0|req: 4/14] 127.0.0.1 () {44 vars in 777 bytes} [Sun May 31 06:51:07 2020] GET /carts/simple/ => generated 803 bytes in 6 msecs (HTTP/1.1 200) 6 headers in 216 bytes (1 switches on core 1)
[pid: 37629|app: 0|req: 5/15] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 06:51:42 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 1 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 37632|app: 0|req: 5/16] 127.0.0.1 () {46 vars in 841 bytes} [Sun May 31 06:51:42 2020] POST /meiduo_admin/authorizations/ => generated 212 bytes in 69 msecs (HTTP/1.1 200) 7 headers in 243 bytes (1 switches on core 0)
[pid: 37629|app: 0|req: 6/17] 127.0.0.1 () {46 vars in 817 bytes} [Sun May 31 06:51:42 2020] OPTIONS /meiduo_admin/statistical/total_count/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 1)
[pid: 37630|app: 0|req: 2/18] 127.0.0.1 () {46 vars in 821 bytes} [Sun May 31 06:51:42 2020] OPTIONS /meiduo_admin/statistical/day_increment/ => generated 0 bytes in 1 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 37629|app: 0|req: 7/19] 127.0.0.1 () {46 vars in 825 bytes} [Sun May 31 06:51:42 2020] OPTIONS /meiduo_admin/statistical/month_increment/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 1)
[pid: 37629|app: 0|req: 8/20] 127.0.0.1 () {46 vars in 815 bytes} [Sun May 31 06:51:42 2020] OPTIONS /meiduo_admin/statistical/day_active/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 1)
[pid: 37629|app: 0|req: 9/21] 127.0.0.1 () {46 vars in 815 bytes} [Sun May 31 06:51:42 2020] OPTIONS /meiduo_admin/statistical/day_orders/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 37629|app: 0|req: 10/22] 127.0.0.1 () {46 vars in 815 bytes} [Sun May 31 06:51:42 2020] OPTIONS /meiduo_admin/statistical/day_active/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 1)
[pid: 37630|app: 0|req: 4/23] 127.0.0.1 () {46 vars in 815 bytes} [Sun May 31 06:51:42 2020] OPTIONS /meiduo_admin/statistical/day_orders/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 1)
[pid: 37630|app: 0|req: 4/24] 127.0.0.1 () {46 vars in 825 bytes} [Sun May 31 06:51:42 2020] OPTIONS /meiduo_admin/statistical/goods_day_views/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.last_login received a naive datetime (2020-05-31 00:00:00) while time zone support is active.
  RuntimeWarning)
[pid: 37629|app: 0|req: 11/25] 127.0.0.1 () {44 vars in 947 bytes} [Sun May 31 06:51:42 2020] GET /meiduo_admin/statistical/day_active/ => generated 31 bytes in 7 msecs (HTTP/1.1 200) 7 headers in 247 bytes (1 switches on core 0)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-31 00:00:00) while time zone support is active.
  RuntimeWarning)
[pid: 37629|app: 0|req: 13/26] 127.0.0.1 () {44 vars in 953 bytes} [Sun May 31 06:51:42 2020] GET /meiduo_admin/statistical/day_increment/ => generated 31 bytes in 6 msecs (HTTP/1.1 200) 7 headers in 247 bytes (1 switches on core 1)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField GoodsVisitCount.create_time received a naive datetime (2020-05-31 00:00:00) while time zone support is active.
  RuntimeWarning)
[pid: 37629|app: 0|req: 13/27] 127.0.0.1 () {44 vars in 947 bytes} [Sun May 31 06:51:42 2020] GET /meiduo_admin/statistical/day_active/ => generated 31 bytes in 8 msecs (HTTP/1.1 200) 7 headers in 247 bytes (1 switches on core 0)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField OrderInfo.create_time received a naive datetime (2020-05-31 00:00:00) while time zone support is active.
  RuntimeWarning)
[pid: 37632|app: 0|req: 7/28] 127.0.0.1 () {44 vars in 957 bytes} [Sun May 31 06:51:42 2020] GET /meiduo_admin/statistical/goods_day_views/ => generated 2 bytes in 10 msecs (HTTP/1.1 200) 7 headers in 246 bytes (1 switches on core 0)
[pid: 37627|app: 0|req: 7/29] 127.0.0.1 () {44 vars in 949 bytes} [Sun May 31 06:51:42 2020] GET /meiduo_admin/statistical/total_count/ => generated 32 bytes in 16 msecs (HTTP/1.1 200) 7 headers in 247 bytes (1 switches on core 0)
[pid: 37627|app: 0|req: 7/30] 127.0.0.1 () {44 vars in 947 bytes} [Sun May 31 06:51:42 2020] GET /meiduo_admin/statistical/day_orders/ => generated 31 bytes in 15 msecs (HTTP/1.1 200) 7 headers in 247 bytes (1 switches on core 1)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-02 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-03 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-04 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-05 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField OrderInfo.create_time received a naive datetime (2020-05-31 00:00:00) while time zone support is active.
  RuntimeWarning)
[pid: 37629|app: 0|req: 14/31] 127.0.0.1 () {44 vars in 947 bytes} [Sun May 31 06:51:42 2020] GET /meiduo_admin/statistical/day_orders/ => generated 31 bytes in 7 msecs (HTTP/1.1 200) 7 headers in 247 bytes (1 switches on core 1)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-06 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-07 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-08 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-09 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-10 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-11 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-12 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-13 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-14 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-15 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-16 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-17 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-18 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-19 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-20 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-21 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-22 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-23 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-24 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-25 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-26 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-27 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-28 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-29 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-30 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-31 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-06-01 00:00:00) while time zone support is active.
  RuntimeWarning)
[pid: 37632|app: 0|req: 7/32] 127.0.0.1 () {44 vars in 957 bytes} [Sun May 31 06:51:42 2020] GET /meiduo_admin/statistical/month_increment/ => generated 961 bytes in 49 msecs (HTTP/1.1 200) 7 headers in 248 bytes (1 switches on core 1)
[pid: 37632|app: 0|req: 8/33] 127.0.0.1 () {46 vars in 819 bytes} [Sun May 31 06:51:45 2020] OPTIONS /meiduo_admin/permission/content_types/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 37630|app: 0|req: 5/34] 127.0.0.1 () {46 vars in 819 bytes} [Sun May 31 06:51:45 2020] OPTIONS /meiduo_admin/permission/content_types/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 1)
[pid: 37630|app: 0|req: 6/35] 127.0.0.1 () {46 vars in 840 bytes} [Sun May 31 06:51:45 2020] OPTIONS /meiduo_admin/permission/perms/?page=1&pagesize=10 => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 37632|app: 0|req: 10/36] 127.0.0.1 () {44 vars in 787 bytes} [Sun May 31 06:51:45 2020] GET /meiduo_admin/permission/content_types/ => generated 38 bytes in 22 msecs (HTTP/1.1 401) 8 headers in 292 bytes (1 switches on core 1)
[pid: 37632|app: 0|req: 11/37] 127.0.0.1 () {44 vars in 787 bytes} [Sun May 31 06:51:45 2020] GET /meiduo_admin/permission/content_types/ => generated 38 bytes in 1 msecs (HTTP/1.1 401) 8 headers in 292 bytes (1 switches on core 1)
[pid: 37632|app: 0|req: 11/38] 127.0.0.1 () {44 vars in 972 bytes} [Sun May 31 06:51:45 2020] GET /meiduo_admin/permission/perms/?page=1&pagesize=10 => generated 719 bytes in 205 msecs (HTTP/1.1 200) 7 headers in 254 bytes (1 switches on core 0)
[pid: 37632|app: 0|req: 12/39] 127.0.0.1 () {46 vars in 805 bytes} [Sun May 31 06:51:49 2020] OPTIONS /meiduo_admin/permission/simple/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 1)
[pid: 37627|app: 0|req: 8/40] 127.0.0.1 () {46 vars in 805 bytes} [Sun May 31 06:51:49 2020] OPTIONS /meiduo_admin/permission/simple/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 37629|app: 0|req: 15/41] 127.0.0.1 () {46 vars in 842 bytes} [Sun May 31 06:51:49 2020] OPTIONS /meiduo_admin/permission/groups/?page=1&pagesize=10 => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 37629|app: 0|req: 16/42] 127.0.0.1 () {44 vars in 773 bytes} [Sun May 31 06:51:49 2020] GET /meiduo_admin/permission/simple/ => generated 38 bytes in 2 msecs (HTTP/1.1 401) 8 headers in 292 bytes (1 switches on core 1)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/rest_framework/pagination.py:198: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'django.contrib.auth.models.Group'> QuerySet.
  paginator = self.django_paginator_class(queryset, page_size)
[pid: 37630|app: 0|req: 7/43] 127.0.0.1 () {44 vars in 773 bytes} [Sun May 31 06:51:49 2020] GET /meiduo_admin/permission/simple/ => generated 38 bytes in 3 msecs (HTTP/1.1 401) 8 headers in 292 bytes (1 switches on core 1)
[pid: 37627|app: 0|req: 9/44] 127.0.0.1 () {44 vars in 974 bytes} [Sun May 31 06:51:49 2020] GET /meiduo_admin/permission/groups/?page=1&pagesize=10 => generated 244 bytes in 78 msecs (HTTP/1.1 200) 7 headers in 254 bytes (1 switches on core 1)
[pid: 37632|app: 0|req: 13/45] 127.0.0.1 () {44 vars in 773 bytes} [Sun May 31 06:51:53 2020] GET /meiduo_admin/permission/simple/ => generated 38 bytes in 1 msecs (HTTP/1.1 401) 8 headers in 292 bytes (1 switches on core 0)
[pid: 37629|app: 0|req: 17/47] 127.0.0.1 () {46 vars in 819 bytes} [Sun May 31 06:51:53 2020] OPTIONS /meiduo_admin/permission/groups/simple/ => generated 0 bytes in 1 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 37627|app: 0|req: 11/47] 127.0.0.1 () {46 vars in 819 bytes} [Sun May 31 06:51:53 2020] OPTIONS /meiduo_admin/permission/groups/simple/ => generated 0 bytes in 1 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 37627|app: 0|req: 11/48] 127.0.0.1 () {46 vars in 842 bytes} [Sun May 31 06:51:53 2020] OPTIONS /meiduo_admin/permission/admins/?page=1&pagesize=10 => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 1)
[pid: 37629|app: 0|req: 18/49] 127.0.0.1 () {44 vars in 773 bytes} [Sun May 31 06:51:53 2020] GET /meiduo_admin/permission/simple/ => generated 38 bytes in 1 msecs (HTTP/1.1 401) 8 headers in 292 bytes (1 switches on core 1)
[pid: 37627|app: 0|req: 12/50] 127.0.0.1 () {44 vars in 787 bytes} [Sun May 31 06:51:53 2020] GET /meiduo_admin/permission/groups/simple/ => generated 38 bytes in 1 msecs (HTTP/1.1 401) 8 headers in 292 bytes (1 switches on core 0)
[pid: 37627|app: 0|req: 14/51] 127.0.0.1 () {44 vars in 787 bytes} [Sun May 31 06:51:53 2020] GET /meiduo_admin/permission/groups/simple/ => generated 38 bytes in 3 msecs (HTTP/1.1 401) 8 headers in 292 bytes (1 switches on core 1)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/rest_framework/pagination.py:198: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'users.models.User'> QuerySet.
  paginator = self.django_paginator_class(queryset, page_size)
[pid: 37627|app: 0|req: 14/52] 127.0.0.1 () {44 vars in 974 bytes} [Sun May 31 06:51:53 2020] GET /meiduo_admin/permission/admins/?page=1&pagesize=10 => generated 270 bytes in 9 msecs (HTTP/1.1 200) 7 headers in 254 bytes (1 switches on core 0)
[pid: 37627|app: 0|req: 15/53] 127.0.0.1 () {46 vars in 805 bytes} [Sun May 31 08:36:02 2020] OPTIONS /meiduo_admin/permission/simple/ => generated 0 bytes in 1 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 1)
[pid: 37629|app: 0|req: 19/54] 127.0.0.1 () {46 vars in 819 bytes} [Sun May 31 08:36:02 2020] OPTIONS /meiduo_admin/permission/groups/simple/ => generated 0 bytes in 1 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 37630|app: 0|req: 8/55] 127.0.0.1 () {46 vars in 819 bytes} [Sun May 31 08:36:02 2020] OPTIONS /meiduo_admin/permission/groups/simple/ => generated 0 bytes in 1 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 37629|app: 0|req: 20/56] 127.0.0.1 () {46 vars in 805 bytes} [Sun May 31 08:36:02 2020] OPTIONS /meiduo_admin/permission/simple/ => generated 0 bytes in 2 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 1)
[pid: 37630|app: 0|req: 9/57] 127.0.0.1 () {46 vars in 842 bytes} [Sun May 31 08:36:02 2020] OPTIONS /meiduo_admin/permission/admins/?page=1&pagesize=10 => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 37627|app: 0|req: 16/58] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:09 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 37627|app: 0|req: 17/59] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:09 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 1)
[pid: 37627|app: 0|req: 18/60] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:10 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 1)
[pid: 37629|app: 0|req: 21/61] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:10 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 37632|app: 0|req: 14/62] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:10 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 1 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 37632|app: 0|req: 15/63] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:10 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 1)
[pid: 37632|app: 0|req: 16/64] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:10 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 37632|app: 0|req: 17/65] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:11 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 1)
[pid: 37632|app: 0|req: 18/66] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:18 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 37632|app: 0|req: 19/67] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:19 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 1)
[pid: 37632|app: 0|req: 20/68] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:19 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 37632|app: 0|req: 21/69] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:19 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 1)
[pid: 37627|app: 0|req: 19/70] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:36:23 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 37627|app: 0|req: 20/71] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:37:40 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 1)
[pid: 37629|app: 0|req: 22/72] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:37:41 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 1)
[pid: 37632|app: 0|req: 22/73] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:37:47 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Sun May 31 16:38:22 2020] ***
compiled with version: 7.5.0 on 31 May 2020 05:43:50
os: Linux-4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018
nodename: python
machine: x86_64
clock source: unix
detected number of CPU cores: 4
current working directory: /home/ubuntu/Desktop/meiduo_new/meiduo_mall/meiduo_mall
writing pidfile to uwsgi.pid
detected binary path: /home/ubuntu/.virtualenvs/meiduos_mall/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/Desktop/meiduo_new/meiduo_mall
your processes number limit is 29423
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 192.168.107.188:8001 fd 3
Python version: 3.6.9 (default, Apr 18 2020, 01:56:04)  [GCC 8.4.0]
Set PythonHome to /home/ubuntu/.virtualenvs/meiduos_mall/
Python main interpreter initialized at 0x55b130180170
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x55b130180170 pid: 40277 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 40277)
spawned uWSGI worker 1 (pid: 40279, cores: 2)
spawned uWSGI worker 2 (pid: 40280, cores: 2)
spawned uWSGI worker 3 (pid: 40282, cores: 2)
spawned uWSGI worker 4 (pid: 40284, cores: 2)
[pid: 40282|app: 0|req: 1/1] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:38:31 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 3 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 40280|app: 0|req: 1/2] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:41:35 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 3 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 40282|app: 0|req: 2/3] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:41:36 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 1 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 1)
[pid: 40282|app: 0|req: 3/4] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:41:36 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 40284|app: 0|req: 1/5] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:41:36 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 2 msecs (HTTP/1.1 200) 4 headers in 132 bytes (1 switches on core 0)
[pid: 40282|app: 0|req: 4/6] 127.0.0.1 () {46 vars in 799 bytes} [Sun May 31 08:42:05 2020] OPTIONS /meiduo_admin/authorizations/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 1)
[pid: 40282|app: 0|req: 5/7] 127.0.0.1 () {46 vars in 841 bytes} [Sun May 31 08:42:05 2020] POST /meiduo_admin/authorizations/ => generated 212 bytes in 306 msecs (HTTP/1.1 200) 7 headers in 243 bytes (1 switches on core 0)
[pid: 40282|app: 0|req: 6/8] 127.0.0.1 () {46 vars in 817 bytes} [Sun May 31 08:42:06 2020] OPTIONS /meiduo_admin/statistical/total_count/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 1)
[pid: 40284|app: 0|req: 3/9] 127.0.0.1 () {46 vars in 815 bytes} [Sun May 31 08:42:06 2020] OPTIONS /meiduo_admin/statistical/day_active/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 40284|app: 0|req: 3/10] 127.0.0.1 () {46 vars in 821 bytes} [Sun May 31 08:42:06 2020] OPTIONS /meiduo_admin/statistical/day_increment/ => generated 0 bytes in 1 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 1)
[pid: 40284|app: 0|req: 4/11] 127.0.0.1 () {46 vars in 815 bytes} [Sun May 31 08:42:06 2020] OPTIONS /meiduo_admin/statistical/day_orders/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 40280|app: 0|req: 2/12] 127.0.0.1 () {46 vars in 825 bytes} [Sun May 31 08:42:06 2020] OPTIONS /meiduo_admin/statistical/month_increment/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 1)
[pid: 40280|app: 0|req: 3/13] 127.0.0.1 () {46 vars in 815 bytes} [Sun May 31 08:42:06 2020] OPTIONS /meiduo_admin/statistical/day_active/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 40282|app: 0|req: 7/14] 127.0.0.1 () {46 vars in 825 bytes} [Sun May 31 08:42:06 2020] OPTIONS /meiduo_admin/statistical/goods_day_views/ => generated 0 bytes in 0 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
[pid: 40279|app: 0|req: 1/15] 127.0.0.1 () {46 vars in 815 bytes} [Sun May 31 08:42:06 2020] OPTIONS /meiduo_admin/statistical/day_orders/ => generated 0 bytes in 6 msecs (HTTP/1.1 200) 8 headers in 431 bytes (1 switches on core 0)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField OrderInfo.create_time received a naive datetime (2020-05-31 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-02 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-03 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-04 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-05 00:00:00) while time zone support is active.
  RuntimeWarning)
[pid: 40282|app: 0|req: 9/16] 127.0.0.1 () {44 vars in 947 bytes} [Sun May 31 08:42:06 2020] GET /meiduo_admin/statistical/day_orders/ => generated 31 bytes in 14 msecs (HTTP/1.1 200) 7 headers in 247 bytes (1 switches on core 1)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-06 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-07 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-08 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-09 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-10 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-11 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-12 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-13 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-14 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-15 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-16 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-17 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-18 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-19 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-20 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-21 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-22 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-23 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-24 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-25 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-26 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-27 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-28 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-29 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-30 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-31 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-06-01 00:00:00) while time zone support is active.
  RuntimeWarning)
[pid: 40282|app: 0|req: 9/17] 127.0.0.1 () {44 vars in 957 bytes} [Sun May 31 08:42:06 2020] GET /meiduo_admin/statistical/month_increment/ => generated 961 bytes in 49 msecs (HTTP/1.1 200) 7 headers in 248 bytes (1 switches on core 0)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.date_joined received a naive datetime (2020-05-31 00:00:00) while time zone support is active.
  RuntimeWarning)
[pid: 40280|app: 0|req: 5/18] 127.0.0.1 () {44 vars in 953 bytes} [Sun May 31 08:42:06 2020] GET /meiduo_admin/statistical/day_increment/ => generated 31 bytes in 318 msecs (HTTP/1.1 200) 7 headers in 247 bytes (1 switches on core 0)
[pid: 40280|app: 0|req: 5/19] 127.0.0.1 () {44 vars in 949 bytes} [Sun May 31 08:42:06 2020] GET /meiduo_admin/statistical/total_count/ => generated 32 bytes in 319 msecs (HTTP/1.1 200) 7 headers in 247 bytes (1 switches on core 1)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField OrderInfo.create_time received a naive datetime (2020-05-31 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField GoodsVisitCount.create_time received a naive datetime (2020-05-31 00:00:00) while time zone support is active.
  RuntimeWarning)
/home/ubuntu/.virtualenvs/meiduos_mall/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1365: RuntimeWarning: DateTimeField User.last_login received a naive datetime (2020-05-31 00:00:00) while time zone support is active.
  RuntimeWarning)
[pid: 40284|app: 0|req: 6/20] 127.0.0.1 () {44 vars in 947 bytes} [Sun May 31 08:42:06 2020] GET /meiduo_admin/statistical/day_orders/ => generated 31 bytes in 346 msecs (HTTP/1.1 200) 7 headers in 247 bytes (1 switches on core 0)
[pid: 40284|app: 0|req: 6/21] 127.0.0.1 () {44 vars in 957 bytes} [Sun May 31 08:42:06 2020] GET /meiduo_admin/statistical/goods_day_views/ => generated 2 bytes in 351 msecs (HTTP/1.1 200) 7 headers in 246 bytes (1 switches on core 1)
[pid: 40279|app: 0|req: 2/22] 127.0.0.1 () {44 vars in 947 bytes} [Sun May 31 08:42:06 2020] GET /meiduo_admin/statistical/day_active/ => generated 31 bytes in 360 msecs (HTTP/1.1 200) 7 headers in 247 bytes (1 switches on core 1)
[pid: 40279|app: 0|req: 3/23] 127.0.0.1 () {44 vars in 947 bytes} [Sun May 31 08:42:06 2020] GET /meiduo_admin/statistical/day_active/ => generated 31 bytes in 5 msecs (HTTP/1.1 200) 7 headers in 247 bytes (1 switches on core 0)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Sun May 31 21:32:27 2020] ***
compiled with version: 7.5.0 on 31 May 2020 05:43:50
os: Linux-4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018
nodename: python
machine: x86_64
clock source: unix
detected number of CPU cores: 4
current working directory: /home/ubuntu/Desktop/meiduo_new/meiduo_mall/meiduo_mall
writing pidfile to uwsgi.pid
detected binary path: /home/ubuntu/.virtualenvs/meiduos_mall/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/Desktop/meiduo_new/meiduo_mall
your processes number limit is 29423
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 192.168.107.188:8001 fd 3
Python version: 3.6.9 (default, Apr 18 2020, 01:56:04)  [GCC 8.4.0]
Set PythonHome to /home/ubuntu/.virtualenvs/meiduos_mall/
Python main interpreter initialized at 0x55c326616170
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x55c326616170 pid: 70805 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 70805)
spawned uWSGI worker 1 (pid: 70807, cores: 2)
spawned uWSGI worker 2 (pid: 70808, cores: 2)
spawned uWSGI worker 3 (pid: 70810, cores: 2)
spawned uWSGI worker 4 (pid: 70811, cores: 2)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 2 seconds
worker 2 buried after 2 seconds
worker 3 buried after 2 seconds
worker 4 buried after 2 seconds
goodbye to uWSGI.
